src/core/services/convert.spec.js

```diff
@@ -0,0 +1,81 @@
+import * as Convert from './convert';
+import {identity} from '../utility';
+
+describe('Convert', () => {
+	const testArray = [1, 3, 6, 9];
+	const notPrimitiveArray = [new Map(), new Set()];
+	const parseText = value => value;
+	const parseNumber = value => value;
+	const parseDate = value => value;
+	const parseBool = value => value;
+
+	describe('parseFactory', () => {
+		it('should return parseText func if argument is text', () => {
+			expect((Convert.parseFactory('text')).name).to.be.equals(parseText.name);
```

Comment:
Just minor improvement:
.to, .be chains were created to improve the readability of test cases.
Instead of 'to be equals' you can type 'to be equal' or 'equal'